docs/web-component-samples/app.js

```diff
@@ -35,6 +37,8 @@ function loginSuccess(){
     console.log('Agent login has been succesful');
     ccUserState.classList.remove('disabled');
     widgetsContainer.appendChild(ccUserState);
+    widgetsContainer.appendChild(ccIncomingTask);
+    widgetsContainer.appendChild(ccTaskList);
```

Comment:
Can we have the `onAccepted`, `onDeclined`, `onTaskAccepted`, and `onTaskDeclined` callbacks setup in the WC samples also?